tests/fast-integration/skr-test/skr-test.js

```diff
@@ -68,6 +80,29 @@ describe("SKR test", function() {
   it("order.created.v1 event should trigger the lastorder function", async function () {
     await sendEventAndCheckResponse();
   });
+
+  it("Deletes the resources that have been created", async function () {
+    await deleteMockTestFixture("mocks", testNS);
+  });
+
+  // Check audit log for AWS
+  if (process.env.KEB_PLAN_ID == AWS_PLAN_ID) {
+    it ("Check audit logs", async function() {
+      const groups = [
+        { "resName": "commerce-binding", "groupName": "servicecatalog.k8s.io", "action": "create" },
+        { "resName": "commerce-binding", "groupName": "servicecatalog.k8s.io", "action": "delete" },
+        { "resName": "lastorder", "groupName": "serverless.kyma-project.io", "action": "create" },
+        { "resName": "lastorder", "groupName": "serverless.kyma-project.io", "action": "delete" },
+        {"resName":"commerce-mock", "groupName": "deployments", "action": "create"},
+        {"resName":"commerce-mock", "groupName": "deployments", "action": "delete"}
+      ]
```

Comment:
Please move this to the `checkAuditLogs` function or if it needs to be a parameter import it from some external file and just pass the variable. Now it is a bit too messy in the test scenario.